protocol/contracts/v2/safeGuards/SafeForked.sol

```diff
@@ -10,32 +10,60 @@ import { ISignatureValidator } from "@gnosis.pm/safe-contracts/contracts/interfa
 //      https://forum.safe.global/t/safe-contract-v1-5-0/3383
 library SafeForked {
 
-    // Cloned from ISignatureValidatorConstants
     // bytes4(keccak256("isValidSignature(bytes,bytes)")
+    // Forked from https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/interfaces/ISignatureValidator.sol#L6
     bytes4 internal constant EIP1271_MAGIC_VALUE = 0x20c13b0b;
 
-    // Cloned from SignatureDecoder
-    function signatureSplit(bytes memory signatures, uint256 pos) internal pure returns (uint8 v, bytes32 r, bytes32 s) {
-        // slither-disable-start assembly
+    /// @notice divides bytes signature into `uint8 v, bytes32 r, bytes32 s`.
+    /// @dev Make sure to peform a bounds check for @param pos, to avoid out of bounds access on @param signatures
+    /// Forked from Safe https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/common/SignatureDecoder.sol#L11
+    /// @param pos which signature to read. A prior bounds check of this parameter should be performed, to avoid out of bounds access
+    /// @param signatures concatenated rsv signatures
+    function signatureSplit(bytes memory signatures, uint256 pos)
```

Comment:
The changes here are mostly cosmetic - re-copied from the v1.3.0 branch:
 https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts